src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -177,7 +178,7 @@ static ncclResult_t mscclSchedulerInit() {
   if (useInternalScheduler) {
     NCCLCHECK(mscclInternalSchedulerInit());
   } else {
-    NCCLCHECK(status.mscclSchedulerPtr->init());
+    NCCLCHECK(status.mscclSchedulerPtr->init(comm));
```

Comment:
Add one more argument that passes bootstrap function pointers

Reply:
done.